examples/config/config_parser.py

```diff
@@ -0,0 +1,31 @@
+import yaml
+import platform
+
+print(platform.system())
```

Comment:
Probably won't want this in here, as I believe this will get called whenever this module gets imported

Reply:
Hey man I changed it.